src/page-racial-disparities/PageRacialDisparities.js

```diff
@@ -0,0 +1,543 @@
+    correctionsPopulationRate,
+    proportionIncarcerated,
+    proportionSupervision,
+    proportionIncarceratedOverall,
+    proportionSupervisionOverall,
+    paroleRate,
+    prisonPopulationRate,
+    supervisionMetrics: {
+      [SUPERVISION_TYPES.parole]: parole,
+      [SUPERVISION_TYPES.probation]: probation,
+      [TOTAL_KEY]: supervision,
+    },
+    ftrPopulationRate,
+    pretrialPopulationRate,
+  };
+}
+
+const getCorrectionsRate = (record) => {
+  const correctionsPop = getCorrectionsPopulation(record);
+  const statePop = record.total_state_population;
+  return correctionsPop / statePop;
+};
+
+function getOverallMetrics(data) {
+  // these metrics are not group dependent, they're always the same
+  const black = data.find(matchRace(RACES.black));
+  const white = data.find(matchRace(RACES.white));
+  const nativeAmerican = data.find(matchRace(RACES.nativeAmerican));
+  const hispanic = data.find(matchRace(RACES.hispanic));
+  const total = data.find(matchRace(TOTAL_KEY));
+
+  const totalCorrectionsRate = getCorrectionsRate(total);
+
+  const whiteCorrectionsRate = getCorrectionsRate(white);
+
+  const whiteLikelihood = whiteCorrectionsRate / totalCorrectionsRate;
+
+  const comparedToWhite = {
+    black: getCorrectionsRate(black) / totalCorrectionsRate / whiteLikelihood,
+    hispanic:
+      getCorrectionsRate(hispanic) / totalCorrectionsRate / whiteLikelihood,
+    nativeAmerican:
+      getCorrectionsRate(nativeAmerican) /
+      totalCorrectionsRate /
+      whiteLikelihood,
+  };
+
+  return comparedToWhite;
+}
+
+// these are for use in inline dynamic text; not necessarily the same as control labels
+const supervisionTypeText = {
+  [SUPERVISION_TYPES.parole]: "parole",
+  [SUPERVISION_TYPES.probation]: "probation",
+  [TOTAL_KEY]: "supervision",
+};
+
+const supervisionTypeOptions = [
+  { id: TOTAL_KEY, label: "Everyone" },
+  { id: SUPERVISION_TYPES.probation, label: "Probation" },
+  { id: SUPERVISION_TYPES.parole, label: "Parole" },
+];
+
+export default function PageRacialDisparities() {
+  const { apiData, isLoading } = useChartData("us_nd/race");
+  const [category, setCategory] = useState(RACES.black);
+  const [supervisionType, setSupervisionType] = useState(TOTAL_KEY);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  const metrics = getMetricsForGroup(apiData.racial_disparities, category);
+  const supervisionTypeMetrics = metrics.supervisionMetrics[supervisionType];
+  const overallMetrics = getOverallMetrics(apiData.racial_disparities);
+
+  const ethnonym = ETHNONYMS[category];
+
+  const TITLE = "Racial Disparities";
+  const DESCRIPTION = (
+    <>
+      <p>
+        In North Dakota, people of color are overrepresented in prison, on
+        probation, and on parole.
+      </p>
+      <p>
+        Black North Dakotans are{" "}
+        <DynamicText>{formatDecimal(overallMetrics.black)}</DynamicText> times
+        as likely to be under DOCR control as their white counterparts, Latino
```

Comment:
Latinx maybe? Or Latino and Latina? Not sure if this has already been workshopped.

Reply:
all the wording on this page is language I got from @kbinney and @hobuobi ... I would rather we did a big pass on copy tweaks once they can view this page live instead of ad hoc line edits but I'm flagging this one for them

Reply:
Fair enough!